<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DoubleListWidgetForm</class>
 <widget class="QWidget" name="DoubleListWidgetForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>314</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1">
     <item>
      <widget class="QLabel" name="m_leftItemsLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="m_leftListWidget"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,0,0,0,1">
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="m_addToolButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_addAllToolButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_removeToolButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_removeAllToolButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
     <item>
      <widget class="QLabel" name="m_rightItemsLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="m_rightListWidget"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_leftListWidget</tabstop>
  <tabstop>m_addToolButton</tabstop>
  <tabstop>m_addAllToolButton</tabstop>
  <tabstop>m_removeToolButton</tabstop>
  <tabstop>m_removeAllToolButton</tabstop>
  <tabstop>m_rightListWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
